src/lib.rs

```diff
@@ -321,7 +317,7 @@ where
     ) -> Result<Self, error::NewSessionError> {
         Self::with_capabilities_and_connector(
             webdriver,
-            webdriver::capabilities::Capabilities::new(),
+            &mut webdriver::capabilities::Capabilities::new(),
```

Comment:
Shouldn't this just be `&`?

Reply:
`Session` constructor modifies the capabilities so somewhere it would need to be `&mut Capabilities` from what I can see

Reply:
Ah, I would just have the `Session` constructor do:
```rust
let mut capabilities = cap.to_owned();
```
internally